src/components/ImportAccountsDrawer/ImportAccountsDrawer.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  Button,
+  Drawer,
+  DrawerBody,
+  DrawerCloseButton,
+  DrawerContent,
+  DrawerFooter,
+  DrawerHeader,
+  DrawerOverlay,
+  forwardRef,
+  Skeleton,
+  Switch,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Tooltip,
+  Tr,
+} from '@chakra-ui/react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { type AccountId, fromAccountId } from '@shapeshiftoss/caip'
+import type { Asset } from '@shapeshiftoss/types'
+import { useIsFetching, useQueryClient } from '@tanstack/react-query'
+import { useCallback, useEffect, useMemo, useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { reactQueries } from 'react-queries'
+import { Amount } from 'components/Amount/Amount'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { RawText, Text } from 'components/Text'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { accountIdToLabel } from 'state/slices/portfolioSlice/utils'
+import {
+  selectFeeAssetByChainId,
+  selectPortfolioCryptoPrecisionBalanceByFilter,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+export type ImportAccountsDrawerProps = {
+  chainId: ChainId
```

Comment:
Not a blocker for this PR, but if we're going with the "import all EVM accounts under the Ethereum umbrella" (same we're doing currently with Ledger), then the "ETH"  case will need the be specifically handled to import accounts for all EVM chains (but only display native balance in ETH)

Reply:
sounds good, all of this logic has already changed in the next PR so we can discuss once we have a more tangible implementation 🍻